public/locales/en/translation.json

```diff
@@ -26,7 +26,8 @@
     "high": "high"
   },
   "alert": {
-    "fetch-data": "Data was downloaded"
+    "fetch-data": "Data was downloaded",
+    "add-task": "Task was added"
```

Comment:
Present perfect should be used for informing users about the progress. E.g. The task has been added